ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.txt

```diff
@@ -179,12 +187,235 @@ hideall "--sync--"
+3156.8 "Ascalon's Might" #sync / 1[56]:[^:]*:King Thordan:63C5:/
+# Enrage?
+
+
+# Phase 6: Nidhogg v2
+3500.0 "--sync--" sync / 1[56]:[^:]*:King Thordan:6667:/ window 500,0
+3535.0 "Great Wyrmsbreath and Dread Wyrmsbreath" sync / 1[56]:[^:]*:Hraesvelgr:6D35:/
+3535.7 "Swirling Blizzard" #sync / 1[56]:[^:]*:Hraesvelgr:6D38:/
+3535.8 "Ice Breath" #sync / 1[56]:[^:]*:Hraesvelgr:6D37:/
+3535.8 "Holy Breath" #sync / 1[56]:[^:]*:Hraesvelgr:6D3C:/
+3535.8 "Flame Breath" #sync / 1[56]:[^:]*:Nidhogg:6D36:/
+3535.8 "Staggering Breath" #sync / 1[56]:[^:]*:Nidhogg:6D3D:/
+3543.0 "Mortal Vow" sync / 1[56]:[^:]*:Nidhogg:6D30:/
+3554.3 "Akh Afah" sync / 1[56]:[^:]*:Hraesvelgr:6D42:/
+3567.4 "Hallowed Wings" sync / 1[56]:[^:]*:Hraesvelgr:6D24:/
+3568.5 "Cauterize" sync / 1[56]:[^:]*:Nidhogg:6D3E:/
+3568.6 "Hallowed Plume and Hallowed Wings" #sync / 1[56]:[^:]*:Hraesvelgr:6D29:/
+3577.0 "Mortal Vow" sync / 1[56]:[^:]*:Nidhogg:6D31:/
+3582.4 "Wroth Flames" sync / 1[56]:[^:]*:Nidhogg:6D45:/
+3593.2 "Akh Morn 1" sync / 1[56]:[^:]*:Nidhogg:6D46:/
+3593.3 "Cauterize" #sync / 1[56]:[^:]*:Hraesvelgr:6D3F:/
+3594.9 "Akh Morn 2" sync / 1[56]:[^:]*:Nidhogg:6D47:/
+3596.5 "Akh Morn 3" sync / 1[56]:[^:]*:Nidhogg:6D47:/
+3598.1 "Akh Morn 4" sync / 1[56]:[^:]*:Nidhogg:6D47:/
+3605.2 "Hot Wing" sync / 1[56]:[^:]*:Nidhogg:6D2B:/
+3606.3 "Hot Wing" sync / 1[56]:[^:]*:Nidhogg:6D2C:/
+3607.2 "Spreading Flames and Entangled Flames" sync / 1[56]:[^:]*:Nidhogg:742B:/
+3611.1 "Mortal Vow" sync / 1[56]:[^:]*:Nidhogg:6D31:/
+3623.4 "Akh Afah" sync / 1[56]:[^:]*:Hraesvelgr:6D42:/
+3635.4 "Hallowed Wings" sync / 1[56]:[^:]*:Hraesvelgr:6D26:/
+3635.4 "Hot Tail" #sync / 1[56]:[^:]*:Nidhogg:6D2D:/
+3636.2 "Hallowed Plume and Hallowed Wings" sync / 1[56]:[^:]*:Hraesvelgr:6D29:/
+3636.4 "Hot Tail" #sync / 1[56]:[^:]*:Nidhogg:6D2E:/
+3645.1 "Mortal Vow" sync / 1[56]:[^:]*:Nidhogg:6D31:/
+3655.1 "Great Wyrmsbreath and Dread Wyrmsbreath" sync / 1[56]:[^:]*:Hraesvelgr:6D35:/
+3655.8 "Swirling Blizzard" sync / 1[56]:[^:]*:Hraesvelgr:6D38:/
+3655.9 "Ice Breath" #sync / 1[56]:[^:]*:Hraesvelgr:6D37:/
+3655.9 "Holy Orb" #sync / 1[56]:[^:]*:Hraesvelgr:6D3A:/
+3655.9 "Flame Breath" #sync / 1[56]:[^:]*:Nidhogg:6D36:/
+3655.9 "Dark Orb" #sync / 1[56]:[^:]*:Nidhogg:6D39:/
+3668.4 "Cauterize" sync / 1[56]:[^:]*:Hraesvelgr:6D3F:/
+3674.5 "--sync--" sync / 1[56]:[^:]*:Hraesvelgr:6D40:/
+3675.4 "Touchdown" sync / 1[56]:[^:]*:(Hraesvelgr|Nidhogg):70E7:/
+3679.1 "Mortal Vow" sync / 1[56]:[^:]*:Nidhogg:6D31:/
+3701.8 "Revenge of the Horde (enrage)" sync / 1[56]:[^:]*:(Hraesvelgr|Nidhogg):6D21:/
+
+3711.8 "Resentment?"
+3718.8 "Shockwave?" #sync / 1[56]:[^:]*:(King Thordan|Nidhogg):71E4:/
+3744.6 "Alternate End?" #sync / 1[56]:[^:]*:Dragon-king Thordan:7438:/
+
+
+# Phase 6.5: Eyes v2
+# Note: Shockwave could come from King Thordan or Nidhogg
+3809.5 "--sync--" sync / 1[56]:[^:]*:Right Eye:717A:/ window 200,0
+3809.5 "Resentment"
+3826.5 "Shockwave?" #sync / 1[56]:[^:]*:(King Thordan|Nidhogg):71E4:/
```

Comment:
I'm a little confused why shockwave is listed three times here.  Should 3711.8-3744.6 get removed?

Reply:
I added them to be able to see them in the timeline as players approach Revenge of the Horde. Should the enrage be skipped entirely, it should jump to 3809.5.

Then should the player have chosen to spare King thordan in P5, it will jump to 4000.0, if not, well I don't think there will  be a Shockwave to sync to as King Thordan does not appear (we know this because world second went and tried this out and released a VOD of it)... there are some more spells that happen as it just loops to the Eyes in Nidhogg/Phase 4.

Reply:
For reference: https://twitter.com/takapast/status/1522008634158649345

Reply:
For loops and such where a jump may occur, I've been following track ~50s ahead, but don't sync,